### PR TITLE
Feature: operations per second metric

### PR DESCRIPTION
**Pull request by popravich**
Implements #55 
* [x] code written
* [x] tests updated
* [x] changelog updated
* [x] authors updated

### CODE REVIEW

File: src/pytest_benchmark/stats.py [hunk 1]
```diff
@@ -162,6 +162,12 @@ def iqr_outliers(self):
     def outliers(self):
         return "%s;%s" % (self.stddev_outliers, self.iqr_outliers)
 
+    @cached_property
+    def ops(self):
+        if self.mean:
```
Comment (ionelmc) [inline src/pytest_benchmark/stats.py]: I wonder if `len(self.data)/sum(self.data)` would be more precise. Thoughts?

**Comment by ionelmc** [comment]
About the changelog part: would be very useful to explain what sort of usecases this new metric would serve.

File: src/pytest_benchmark/stats.py (hunk 1 above)
Comment (popravich) [inline src/pytest_benchmark/stats.py]: Yes, true. Updated

**Comment by popravich** [comment]
Added few words in changelog. Please take a look and say if its okay.

**Comment by ionelmc** [comment]
Yes, very good. Lets wait for the slow CI to complete.

**Comment by popravich** [comment]
Thanks

**Comment by ionelmc** [comment]
But I'm supposed to say thanks, thanks!

**Pull request merged**